docs/commands/status.md

```diff
@@ -34,3 +34,40 @@ The following data is reported for each dependency when the YAML or JSON report
 - license: the dependency's SPDX license identifier
 - filename: the full path on disk to the dependency's cached metadata file, if available
 - errors: any error messages from failed status checks, if available
+
+## Status errors and resolutions
+
+### cached dependency record not found
+
+**Cause:** A dependency was found while running `licensed status` that does not have a corresponding cached metadata file
+**Resolution:** Run `licensed cache` to update the metadata cache and create the missing metadata file
+
+### cached dependency record out of date
+
+**Cause:** A dependency was found while running `licensed status` with a different version than is contained in the dependency's cached metadata file
+**Resolution:** Run `licensed cache` to update the out-of-date metadata files
+
+### missing license text
+
+**Cause:** A license file was found or a license determination was otherwise made, but the files contained no usable text
```

Comment:
I'd suggest

`**Cause:** A license determination was made, e.g. from package metadata, but no license text was found`

as I don't think a license determination would be made if a license text was found that contained nothing or otherwise didn't match a license.